docs/github.md

```diff
@@ -76,6 +76,12 @@ commits to your branch[^1]. Some projects (such as Jujutsu and LLVM) instead
 prefer that you keep your commits clean by rewriting them and then
 force-pushing[^2].
 
+[^1]: This is a GitHub Style review, as GitHub currently only is able to compare
+branches.
+
+[^2]: If you're wondering why we prefer clean commits in this project, see
+e.g. [this blog post][stacked]
+
```

Comment:
Optional: Move all links to EOF?

Reply:
Thanks! This made a difference with `mdbook`, but there's no point to this change anymore.